pyee/__init__.py

```diff
@@ -53,28 +53,96 @@ def __init__(self):
 __all__ = ["BaseEventEmitter", "EventEmitter", "PyeeException"]
 
 try:
-    from pyee._asyncio import AsyncIOEventEmitter  # noqa
+    from pyee.asyncio import AsyncIOEventEmitter as BaseAsyncIOEventEmitter  # noqa
+
+    class AsyncIOEventEmitter(BaseAsyncIOEventEmitter):
+        """
+        pyee.AsyncIOEventEmitter is deprecated and an alias for
+        pyee.asyncio.AsyncIOEventEmitter.
+        """
+
+        def __init__(self, loop=None):
+            warn(
+                DeprecationWarning(
+                    "pyee.AsyncIOEventEmitter is deprecated and will be removed "
+                    "in a future major version; you should instead use "
+                    "pyee.asyncio.AsyncIOEventEmitter."
+                )
+            )
+
+            super(AsyncIOEventEmitter, self).__init__(loop=loop)
 
     __all__.append("AsyncIOEventEmitter")
 except ImportError:
-    pass
+    raise
```

Comment:
This is a holdover from testing and needs to be "pass". I'm wondering though if there's a way I can introduce simple debug logging a la logref?